src/main/kotlin/app/revanced/cli/command/PatchCommand.kt

```diff
@@ -90,6 +90,28 @@ internal object PatchCommand : Runnable {
         }
     }
 
+    @ArgGroup(exclusive = false, multiplicity = "0..1")
```

Comment:
Why were these options moved above the "exclusive" option?

Reply:
Unsure what do you mean here. As in why the `-i` & `--mount` blocks were moved up? No particular reason, can move back below.

Reply:
Please move them down.